src/middlewares/sendEmail.js

```diff
@@ -13,6 +13,7 @@ export const sendVerificationEmail = async (req, res, next) => {
   const userInfo = {
     email: email,
     subject: 'Verify your email',
+    html: `<p>Welcome to Barefoot Nomad, Click on the link below to verify your email.</p> <br> <a href='${process.env.FRONTEND_URL}/user/verification?token=${accessToken}'>Link</a>`,
```

Comment:
When using template literals ( ` ...` ), everything inside is a string, string literals will therefore appear as quotes. Please remove any quotes inside the backticks(` `), the URLamsure doesn't contain quotes

Reply:
thanks @Anguandia  i just corrected it